confgenerator/logging_receivers.go

```diff
@@ -682,6 +682,61 @@ func (r LoggingReceiverSystemd) Components(ctx context.Context, tag string) []fl
 	return input
 }
 
+func (r LoggingReceiverSystemd) Pipelines(ctx context.Context) ([]otel.ReceiverPipeline, error) {
+	operators := []map[string]any{}
+	operators = append(operators, map[string]any{
```

Comment:
I think we should use `ModifyFields` here instead of using the deprecated `operators` option to do these manipulations.

Reply:
Yes, using `ModifyFields` is better. I was having a hard time trying to use the `operators` and with `ModifyFields` it was a lot simpler. Done.